apps/static/weblab/bramblehost.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * JS to communicate between Bramble and Code Studio
+ */
+
+(function () {
```

Comment:
I'm going to see if I can find a good way to get this file into our pipeline. at a minimum the indentation in this file is wrong. If we leave it here, it's probably worth making sure it at least gets linted (I suspect it currently does not in this location)